client/src/components/layout/services/Fetcher.js

```diff
@@ -0,0 +1,101 @@
+import _ from "lodash"
+import translateServerErrors from "./translateServerErrors.js"
+/**
+ * A helper class for fetch api:
+ * - static method `.get`
+ * - static method `.post` 
+ */
+class Fetcher {
+  /**
+   *
+   * @param {string} route The domain relative url of the post route
+   * @param {object} body The date object to be stringified in the fetch request
+   * @param {object=} options Options for configuring the post request:
+   * - `validationStatus`: the validation errors status code, default 422
+   * - `validationErrorParser`: parsing function used on `validationErrors' - defaults to parse "objection" validation errors
+   * @returns A response object with properties of :
+   * - `ok`: true if successfully inserted
+   * - `data`: the post response body
+   * - `validationErrors`: the parsed json validation errors
+   */
```

Comment:
I love this markup - but can we please remove it from this codebase so we have consistency across files?